src/braket/aws/aws_device.py

```diff
@@ -47,6 +47,50 @@ class AwsDeviceType(str, Enum):
     QPU = "QPU"
 
 
+class BraketDevices:
```

Comment:
I think this is a nice QOL change, but this does mean that we'll need to manually maintain this enum. Is that scalable long term?

Reply:
Yes, it seems pretty light to maintain. I'll make sure to update the device onboarding template to include this update

Reply:
Can we create this device list dynamically? What do you think of something like `Devices = Enum('Devices',[(device.name, device.arn) for device in AwsDevice.get_devices()])`?

Reply:
I bet we could, but idk how I feel about API calls at import time, plus I don't think that would enable IDE autocomplete.

The other complexity is that not all "device names" are valid python variable names (multiple words, dashes, etc.), so we'd need to have some kind of mapping anyways (could be done by regex, but this seems like something we want manual quality control over)

Reply:
I think it's fine to have this as long as the critical path doesn't rely on its contents, the way it used to; `AwsDevice` used to assign boto3 session regions by hardcoding them in a dict, which made it a pain to add new devices.